### PR TITLE
Added simple shop service

### PR DESCRIPTION
**Pull request by DmtrVoloshyn**

### CODE REVIEW

File: Homework/OnlineStore/Entities/ProductEntity.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Entities
+{
+	public class ProductEntity
+	{
+        public int ProductId { get; set; }
+        public string? ProductName { get; set; }
+        public double Price { get; set; }
+    }
```
Comment (PeterChernykh) [inline Homework/OnlineStore/Entities/ProductEntity.cs:8]: Need to fix style

File: Homework/OnlineStore/Enums/ComputerOperatingSystem.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Enums
+{
+	public enum ComputerOperatingSystem
+	{
+		windows,
+		macOs,
+		unspecified
```
Comment (PeterChernykh) [inline Homework/OnlineStore/Enums/ComputerOperatingSystem.cs]: It's better to use UpperCase

File: Homework/OnlineStore/Models/Smartphone.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using Entities;
+using Enums;
+
+namespace Models
+{
+	public class Smartphone : Device
+	{
+		public PhoneOperatingSystem OperatingSystem { get; set; }
+        public int Memory { get; set; }
```
Comment (PeterChernykh) [inline Homework/OnlineStore/Models/Smartphone.cs:8]: style

File: Homework/OnlineStore/Services/ShopService.cs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                }
+
+            } while (ShouldContinueShopping());
+
+            customerSelection = customerSelections.ToArray();
+
+            _cartProcessingService.CreateCart(customerSelection, cartId);
+        }
+
+        private static bool ShouldContinueShopping()
```
Comment (PeterChernykh) [inline Homework/OnlineStore/Services/ShopService.cs]: IsContinueShopping

**Comment by PeterChernykh** [comment]
Very good. Please solve the conflicts.

File: Homework/OnlineStore/Models/Smartphone.cs (hunk 3 above)
Comment (DmtrVoloshyn) [reply Homework/OnlineStore/Models/Smartphone.cs:8]: hm... locally everything is ok

File: Homework/OnlineStore/Entities/ProductEntity.cs (hunk 1 above)
Comment (DmtrVoloshyn) [reply Homework/OnlineStore/Entities/ProductEntity.cs:8]: locally everything is ok

**Pull request merged**